learn/getting_started/supported_os.md

```diff
@@ -0,0 +1,21 @@
+# Supported operating systems
+
+This page lists operating systems Meilisearch officially supports and tests with every new release. Meilisearch binaries might still run in unsupported environments. Refer to the [changelog](https://github.com/meilisearch/MeiliSearch/releases) for more information on operating system support changes.
+
+## Linux
+
+The Meilisearch binary works on all Linux distributions with `amd64/x86_64` or `aarch64/arm64` architecture using glibc 2.27 and later. You can check your glibc version using:
+
+```
+ldd --version
+```
+
+## macOS
+
+The Meilisearch binary works with macOS 12 and later with `amd64` or `arm64` architecture.
+
+## Windows
+
+The Meilisearch binary works with Windows 11 and later.
+
+If the provided [binaries](/learn/getting_started/quick_start.md#local-installation) don't work for you, try building Meilisearch from source. If compilation fails, Meilisearch is not compatible with your machine.
```

Comment:
Ok, I should have tested this on the preview before making my suggestion. Now it looks like it's part of the Windows instructions. I think the simplest solution is to return it to where it was, right below the first paragraph. Sorry 😅